release-team/role-handbooks/release-notes/README.md

```diff
@@ -230,10 +230,10 @@ Begin running release-notes tool for the ongoing collection of release notes wit
 - Update the TODOs section at the end of each release for the next Release Notes Team
 
 ## Tools
-- [krel](https://github.com/kubernetes/release/tree/master/docs/krel) The Kubernetes Release Toolbox
+- [krel](https://github.com/kubernetes/release/tree/master/docs/krel) The Kubernetes Release Toolbox *(always use the latest version of krel to ensure you have the latest fixes/patches)*
 - [The krel `release-notes` subcommand](https://github.com/kubernetes/release/blob/master/docs/krel/release-notes.md)
 - [The old release notes tool](https://github.com/kubernetes/release/tree/master/cmd/release-notes)
-- [Release notes website](https://relnotes.k8s.io)
+- [Release notes website](https://relnotes.k8s.io) *(release notes website is only required for the final release)*
```

Comment:
```suggestion
- [Release notes website](https://relnotes.k8s.io) *(Release notes website is only required for the final release)*
```

Reply:
Based on the convention in the page, text in brackets seems to start with lower case.
@npolshakova please can you confirm?

Reply:
hi Amit @Checksumz, in case the convention is confirmed to be uppercase, then we need to align the beginning of this modification, which was uppercase. I'm happy to follow up on this with you once confirmed 
![Screen Shot 2024-07-18 at 13 35 02 PM](https://github.com/user-attachments/assets/e3f15904-a383-416b-99da-522f0c6eb3d9)

Reply:
Yep, we haven't been very consistent in following the convention. Maybe we can add a `note:` to the comment? 

```suggestion
- [Release notes website](https://relnotes.k8s.io) *(note: release notes website is only required for the final release)*
```

That way it's clear that these are suggestions related to the tools? 

<img width="932" alt="image" src="https://github.com/user-attachments/assets/69d495ea-08f9-4eb7-9b81-7d6d35017309">

Reply:
We might also want to add an example on how to check you have the latest krel version/how to upgrade.